### ISSUE TITLE
Add term abnormal finger phalanx ossification

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10439]
**Preferred term label:**
Abnormal finger phalanx ossification

**Synonyms**
Abnormal ossification involving phalangeal bones of the hand, Abnormal ossification of phalanges of the hand

**Definition (free text, please give PubMed ID)**
Any abnormal process of ossification of the phalangeal bones, which normally are ossified from the body and proximal extremity. Ossification of the body commences during the eighth week of fetal development. Ossification of the proximal extremity commences in the bones between the third and fourth years.

Gray H. Anatomy of the human body. Bartleby.com. ISBN:1587341026, 
Hacking C, Gaillard F, Worsley C, et al. Phalanges of the hands. Reference article, Radiopaedia.org (Accessed on 05 Apr 2024) https://doi.org/10.53347/rID-41452

**Parent term (use hpo.jax.org/app)**
Abnormal finger phalanx morphology HP:0005918
Abnormal hand bone ossification HP:0010660

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, Acromesomelic dysplasia type-grebe (omim200700) PMID: 27577507

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000922
- label: Abnormal finger phalanx ossification
- parents: [Abnormal hand bone ossification (HP:0010660)](https://hpo.jax.org/app/browse/term/HP:0010660) | [Abnormal finger phalanx morphology (HP:0005918)](https://hpo.jax.org/app/browse/term/HP:0005918)
- pmid: 
- def: Any abnormal process of ossification of the phalangeal bones, which normally are ossified from the body and proximal extremity. Ossification of the body commences during the eighth week of fetal development. Ossification of the proximal extremity commences in the bones between the third and fourth years.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10439
- synonyms:
